packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js

```diff
@@ -127,6 +128,38 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#extractOrgIdFromJWT()', () => {
+      let credentials;
+      let jwt;
+      let jwtOrgId;
+      let webex;
+
+      beforeEach('generate a JWT and Webex Instance', () => {
+        webex = new MockWebex({
+          children: {
+            credentials: Credentials
+          }
+        });
+
+        credentials = webex.credentials;
+
+        jwt = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJndWVzdC11c2VyLTczNDkiLCJuYW1lIjoiR3Vlc3QgVXNlciBEaXNwbGF5IE5hbWUiLCJpc3MiOiJZMmx6WTI5emNHRnlhem92TDNWekwwOVNSMEZPU1ZwQlZFbFBUaTg1Tm1GaVl6SmhZUzB6WkdOakxURXhaVFV0WVRFMU1pMW1aVE0wT0RFNVkyUmpPV0UiLCJleHAiOiIxNTExMjg2ODQ5In0.xGVs2k-JLGXxZyhMVVGZWlCf5Nu5nz_wT1vaHmH4Etg';
+
+        const fields = jwt.split('.');
+
+        jwtOrgId = JSON.parse(jose.util.base64url.decode(fields[1])).realm;
```

Comment:
Instead of re-writing the logic of what `extractOrgIdFromJWT` does, would it be beneficial to hardcode the expected value instead?

Reply:
I moved this logic to the appropriate place. With integration test users, we get all the information we need to validate this method works as expected.